app/models/concerns/edition/lead_image.rb

```diff
@@ -18,14 +18,6 @@ def high_resolution_lead_image_url
     image_data.file.url(:s960)
   end
 
-  def lead_image_alt_text
```

Comment:
Nitpick: I'd move this to a separate 'lead image alt text removal' commit, rather than lump it in with the 'image alt text removal' commit.

Reply:
Unresolving this thread as the suggestion still stands.